packages/text-field/src/TextField.js

```diff
@@ -98,11 +101,25 @@ export default class TextField extends Component {
     return this.props.onChange && this.props.onChange(event);
   };
 
+  handleInputClear = event => {
+    this.setState({ value: "" });
+    // @TODO: seems we should also trigger the onChange handler?
+    return (
```

Comment:
Same as above; this `return` isn't need, and we should use a regular if statement.